hieradata/site/ls/cluster/lsstcam-ccs.yaml

```diff
@@ -7,3 +7,13 @@ ccs_software::service_email: "base-teststand-alerts-aaaaojwlqmczgqmavrwyopux5a@r
 
 ccs_monit::alert:
   - "base-teststand-alerts-aaaaojwlqmczgqmavrwyopux5a@rubin-obs.slack.com"
+
+s3daemon::instances:
+  ls-lsstcam:
+    s3_endpoint_url: "https://s3.ls.lsst.org"
+    port: 15570
+    image: "%{alias('s3daemon_default_image')}"
+  sdfembs3-lsstcam:
+    s3_endpoint_url: "https://sdfembs3.sdf.slac.stanford.edu"
```

Comment:
I'm not sure we can talk on the IPSec tunnels from the Base.  This may have to be s3dfrgw instead.

Reply:
correct - no route from base to ipsec tunnel

Reply:
@ktlim I wasn't sure what to do for BTS. Obviously, need need a means of testing. Should this be changed to s3dfrgw? One option would be to configure another instance that connects to a local s3 endpoint but listens on a different port in order to allow testing with a "remote push" + "local push" workflow.

Reply:
I'm going to remove sdfembs3 completely for `ls` for the moment. We will need to revisit how testing on BTS should work.